### ISSUE TITLE
Limited method completion suggestions to some reasonable amount?

### ISSUE THREAD

**Issue by jakebolewski** [JuliaLang/julia#13896]
If you tab complete `julia> convert(`, be prepared to scroll _way_ back to see what you were doing before.

**Comment by yuyichao** [comment]
Or do what readline does and ask before printing out the result if there's to many and only print one page at a time.

**Comment by jakebolewski** [comment]
Yes, that would be the :a:⁺ solution

**Comment by StefanKarpinski** [comment]
The A++ solution would be to do what zsh does and print a navigable, scrolling list of options and highlight the currently selected one. It's really slick.

**Comment by jakebolewski** [comment]
Yes that would be nice to have (in general) but for this case I don't think tabbing through ~600 method definitions is really helpful :-)

**Comment by StefanKarpinski** [comment]
Couple with some search mechanism, it could be.
